package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
Yeah, that's the thing. `Material Icons` uses its label name in the command. We use just `Icons` where we should had used `VSCode Icons` or `vscode-icons` if you prefer.